react-renderer/src/Evaluator.ts

```diff
@@ -230,7 +281,7 @@ export const evalExpr = (
       return {
         tag: "Val",
         // HACK: coerce the type for now to let the compiler finish
-        contents: evalUOp(uOp, arg as IFloatV<Tensor> | IIntV<Tensor>),
+        contents: evalUOp(uOp, arg as IFloatV<VarAD> | IIntV<VarAD>),
       }
     } break;
```

Comment:
Sorry about missing these. I don't think the `break;`s do anything here.

Reply:
Forgot to comment but I did fix this earlier